views/index.etlua

```diff
@@ -15,7 +15,18 @@
         <img src="/static/images/github-icon.svg" alt="Github" width="32" class="logo">
     </a>
 </div>
-
+<script>
+  document.getElementById('fengari-input').addEventListener('focus', function() {
```

Comment:
I don't think a placeholder needs to be manually managed

Reply:
The default behavior is that is remains until input. I added the js to draw more attention to the fact that it was an interactive/input element. Do you want me to take it out?